commit 6f7415033813c73fde81da60d209297901f35575
Author: Adhemerval Zanella <azanella@linux.vnet.ibm.com>
Date:   Thu Feb 12 05:34:16 2015 -0600

    powerpc: Fix TABORT encoding for little endian
    
    This patch fix the TABORT encoding for toolchains with no support for
    HTM builtins.

diff --git a/sysdeps/unix/sysv/linux/powerpc/htm.h b/sysdeps/unix/sysv/linux/powerpc/htm.h
index edac601..57d5cd6 100644
--- a/sysdeps/unix/sysv/linux/powerpc/htm.h
+++ b/sysdeps/unix/sysv/linux/powerpc/htm.h
@@ -60,7 +60,7 @@
 #define TBEGIN ".long 0x7c00051d"
 #define TEND   ".long 0x7c00055d"
 #if __BYTE_ORDER == __LITTLE_ENDIAN
-# define TABORT ".byte 0x1d,0x07,%1,0x1d"
+# define TABORT ".byte 0x1d,0x07,%1,0x7c"
 #else
 # define TABORT ".byte 0x7c,%1,0x07,0x1d"
 #endif
